terraform/modules/network-validator/scripts/network_validation.sh

```diff
@@ -1019,6 +1072,39 @@ then
         fi
       fi
     fi
+
+    # In secure production mode, check if Check if bastion Host IP CIDR or Bastion Subnet CIDR has access to Admin HTTPS port on WLS subnet or Admin Server NSG
```

Comment:
Please change "Admin HTTPS Port" to "Administration Port". This is the name of the port in WLS documentation so we should keep the name consistent.

Reply:
Please change secure production mode -> secured production mode